src/style.css

```diff
@@ -1 +1,61 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 20px;
+}
```

Comment:
It is generally not a good idea to fix container size unless there is a specific requirement to do so. Here you have set a width of 100% for the `.search-wrapper` class. Consider letting the content size dictate the container size to avoid overflow or accidental scroll bar.